pkg/crc/machine/start_test.go

```diff
@@ -0,0 +1,127 @@
+package machine
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/containers/common/pkg/strongunits"
+	"github.com/stretchr/testify/assert"
+)
+
+type MockedSSHRunner struct {
+	mockedSSHCommandToOutputMap map[string]string
+	mockedSSHCommandToError     map[string]error
+	mockedSSHCommandToArgsMap   map[string]string
```

Comment:
nit: maybe remove the `Map` suffix from the other two struct members (`mockedSSHCommandToOutputMap`, `mockedSSHCommandToArgsMap`) as well

Reply:
Thanks, let me update it as per your suggestion.